com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/usecases/TestMissingCopybookNotInVariableList.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package com.ca.lsp.cobol.usecases;
+
+import com.broadcom.lsp.cdi.LangServerCtx;
+import com.ca.lsp.cobol.ConfigurableTest;
+import com.ca.lsp.cobol.service.delegates.validations.AnalysisResult;
+import com.ca.lsp.cobol.service.mocks.MockWorkspaceService;
+import org.eclipse.lsp4j.Location;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static com.ca.lsp.cobol.service.delegates.validations.UseCaseUtils.analyze;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/** This test covers case when copybook that was not found appears as a variable. */
+public class TestMissingCopybookNotInVariableList extends ConfigurableTest {
+  private static final String TEXT =
+      "       IDENTIFICATION DIVISION.\n"
+          + "       PROGRAM-ID. TEST1.\n"
+          + "       DATA DIVISION.\n"
+          + "       WORKING-STORAGE SECTION.\n"
+          + "       01 PARENT. COPY CPYNAME.\n"
+          + "       PROCEDURE DIVISION.\n"
+          + "       MOVE 00 TO CHILD1 OF PARENT.";
+
+  public TestMissingCopybookNotInVariableList() {
+
+    MockWorkspaceService workspaceService =
+        LangServerCtx.getInjector().getInstance(MockWorkspaceService.class);
+    workspaceService.setCopybooks(Collections::emptyList);
+  }
+
+  @Test
+  public void test() {
```

Comment:
Pls few lines of description about the test

Reply:
There is a description in the file header